src/components/BooleanInput/index.ts

```diff
@@ -20,19 +24,11 @@ export interface BooleanInputArgs extends ElementArgs, IBindableArgs {
  * A checkbox element.
  */
 class BooleanInput extends Input implements IBindable, IFocusable {
-    static readonly defaultArgs: BooleanInputArgs = {
-        ...Element.defaultArgs,
-        renderChanges: false,
-        value: false,
-        tabIndex: 0,
-        dom: 'div'
-    };
-
     protected _value: boolean;
 
-    constructor(args: BooleanInputArgs = BooleanInput.defaultArgs) {
-        args = { ...BooleanInput.defaultArgs, ...args };
-        super(args.dom, args);
+    constructor(args: BooleanInputArgs = {}) {
+        args.tabIndex = args.tabIndex ?? 0;
+        super(args.dom ?? 'div', args);
```

Comment:
`dom` defaults to `div` (in the `Element` class) so I don't think you need to do this.

Reply:
👍 I've removed all these